content/docs/user-guide/components/reference/paintbrush/paintbrush.md

```diff
@@ -0,0 +1,169 @@
+| 100% | ![Paintbrush flow of 100%.](/images/user-guide/components/reference/paintbrush/paintbrush-flow-100.png) |
+
+Because the Flow opacity affects each brush circle, overlapping circles in the same brush stroke will blend with each other.
+
+An example of a brush stroke crossing itself with medium flow:
+![Example of brush stroke crossing itself with medium flow.](/images/user-guide/components/reference/paintbrush/paintbrush-flow-overlap.png)
+
+#### Hardness
+
+**Hardness** controls the opacity falloff of each brush circle. The Hardness is the percent distance along the radius of the circle to begin the falloff. The falloff always ends at the outer radius. For example, a Hardness value of 50% means that the inner 50% of the brush circle will use the maximum opacity specified by Flow, and the outer 50% of the brush circle will drop in opacity until it is completely transparent. A value of 100% will make the entire brush circle use the maximum opacity.
+
+An illustration of Hardness 50% - the inner circle shows where the falloff begins, and the outer circle shows where the falloff ends. The outer circle is the brush size:![Illustration of hardness opacity falloff.](/images/user-guide/components/reference/paintbrush/paintbrush-falloff-50-percent.png)
+
+The following chart shows how the overall circle opacity changes through different combinations of Hardness and Flow. Although the circles appear to be different sizes, they are all the same brush size with differing amounts of opacity between the center and the outer edge. Lower Flow values have a lower maximum opacity which causes the falloff to reach 0 more quickly.
+
+| | Flow 1% | Flow 50% | Flow 100% |
+| - | - | - | - |
+| Hardness 0% | ![0% Hardness 1% Flow.](/images/user-guide/components/reference/paintbrush/paintbrush-hardness-0-flow-1.png) | ![0% Hardness 50% Flow.](/images/user-guide/components/reference/paintbrush/paintbrush-hardness-0-flow-50.png) | ![0% Hardness 100% Flow.](/images/user-guide/components/reference/paintbrush/paintbrush-hardness-0-flow-100.png) |
+| Hardness 50% | ![50% Hardness 1% Flow.](/images/user-guide/components/reference/paintbrush/paintbrush-hardness-50-flow-1.png) | ![50% Hardness 50% Flow.](/images/user-guide/components/reference/paintbrush/paintbrush-hardness-50-flow-50.png) | ![50% Hardness 100% Flow.](/images/user-guide/components/reference/paintbrush/paintbrush-hardness-50-flow-100.png) |
+| Hardness 100% | ![100% Hardness 1% Flow.](/images/user-guide/components/reference/paintbrush/paintbrush-hardness-100-flow-1.png) | ![100% Hardness 50% Flow.](/images/user-guide/components/reference/paintbrush/paintbrush-hardness-100-flow-50.png) | ![100% Hardness 100% Flow.](/images/user-guide/components/reference/paintbrush/paintbrush-hardness-100-flow-100.png) |
+
+Because the Hardness opacity affects each brush circle, overlapping circles in the same brush stroke will blend with each other.
```

Comment:
```suggestion
Because the **Hardness** opacity affects each brush circle, overlapping circles in the same brush stroke will blend with each other.
```